static/admin/index.html

```diff
@@ -45,8 +45,8 @@ <h1 class="translate">Admin Tools</h1>
     <b><span class="translate">Authentication status</span>: </b>
     <span id="authentication_placeholder"></span>
 
-    <script src="/js/bundle.js?v=0.10.0-dev-20170821"></script>
+    <script src="/js/bundle.js?v=8.1.4-dev-2017082601"></script>
```

Comment:
all the `0.10.0`'s got replaced with `8.1.4`, replacing node versions somewhere else?